tests/test_template.py

```
@@ -1,102 +1,587 @@
+            path_bin / "whitaker"
+            if whitaker_location == "path"
+            else user_bin / "whitaker"
+        )
+        expected_whitaker.write_text(
+            f"#!/bin/sh\ntouch {marker}\nexit 0\n", encoding="utf-8"
+        )
+        expected_whitaker.chmod(0o755)
+    make = shutil.which("make")
+    assert make is not None, "expected make to be available for generated tests"
+
+    result = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)